dataeng/jobs/analytics/PrefectFlowsDeployment.groovy

```diff
@@ -40,8 +40,12 @@ class PrefectFlowsDeployment{
                     }
                 }
             }
+            triggers {
+                pollSCM {
+                    scmpoll_spec(allVars.get('JOB_FREQUENCY'))
+                }
+            }
             publishers common_publishers(allVars)
-            triggers common_triggers(allVars, allVars)
             wrappers common_wrappers(allVars)
```

Comment:
I think it's similar to writing, that I have used in m2m job: https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/jobs/analytics/WarehouseTransformsCIMasterMerges.groovy#L49-L52
```
            triggers {
                scm('H/10 * * * *')
            }
```